js/foundation.offcanvas.js

```diff
@@ -224,38 +224,20 @@ class OffCanvas {
         last = focusable.eq(-1);
 
     focusable.off('.zf.offcanvas').on('keydown.zf.offcanvas', function(e) {
-      var key = Foundation.Keyboard.parseKey(e);
-      if (key === 'TAB' && e.target === last[0]) {
-        e.preventDefault();
-        first.focus();
-      }
-      if (key === 'SHIFT_TAB' && e.target === first[0]) {
-        e.preventDefault();
-        last.focus();
+      if (e.which === 9 || e.keycode === 9) {
```

Comment:
Do we need the extra `e.keycode` instead of just `e.which`?  Also, part of the benefit of the Foundation.Keyboard is it abstracts away the values of the keycodes and lets you have them named, catches combos with shift, etc... was there a reason for going away from using it?